src/resources/packages/classy/fields/EventLocation/VenueUpsert.tsx

```diff
@@ -69,6 +70,7 @@ export default function VenueUpsert( props: {
 
 	// At a minimum, a Venue requires a name.
 	const [ confirmEnabled, setConfirmEnabled ] = useState( currentValues.name !== '' );
+	const [ isUrlInvalid, setIsUrlInvalid ] = useState< boolean >( false );
```

Comment:
Same question about initial value here, should it be based on the venue current values?
And same observation about making the const a positive one, `isValidUrl`.